### PR TITLE
Fix happo flake

### PR DESCRIPTION
**Pull request by BCerki**
card: https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=61334238

This PR:
- refactors .spec file code to use POMs and helper functions
- adds three stabilization functions to help with flakiness
- there were a couple of the screenshots that were flaky no matter how much stabilization I threw at them. I made sure to leave one screenshot in at least one browser everywhere and put notes about this in the docs

### CODE REVIEW

File: bciers/apps/registration/e2e/utils/helpers.ts [hunk 1]
```diff
@@ -447,3 +447,40 @@ export async function waitForElementToStabilize(page: Page, element: string) {
...
+export async function stabilizeAccordion(
+  page: Page,
+  expectedArrowDropdownCount: number,
+) {
+  await waitForElementToStabilize(page, "section");
+  const arrowDropDownElements = await page.locator(
+    `[data-testid=${DataTestID.ARROW_DROPDOWN_ICON}]`,
+  );
+  expect(await arrowDropDownElements.count()).toBe(expectedArrowDropdownCount);
+  await waitForElementToStabilize(page, "section");
```
Comment (BCerki) [inline bciers/apps/registration/e2e/utils/helpers.ts:485]: I've got three assertions here to give the page lots of time to stabilize

Comment (BCerki) [reply bciers/apps/registration/e2e/utils/helpers.ts:486]: Accordions are removed in part II, so this won't be a permanent problem

File: bciers/apps/registration/e2e/workflows/bceidbusiness/industry_user_admin.spec.ts [hunk 2]
```diff
@@ -111,12 +113,12 @@ test.describe("Test Workflow industry_user_admin", () => {
...
     await operationsPage.tableHasExpectedColumns(UserRole.INDUSTRY_USER_ADMIN);
-    await tableHasExpectedRowCount(page, 14);
-    expect(page.locator(".MuiDataGrid-row:hover")).toHaveCount(0);
     // 📷 Cheese!
-    await happoPlaywright.screenshot(operationPage.page, pageContent, {
+    await stabilizeGrid(page, 14);
+    await takeStabilizedScreenshot(happoPlaywright, operationPage.page, {
       component: "Operation grid",
       variant: UserRole.INDUSTRY_USER_ADMIN,
+      targets: ["chrome", "firefox", "safari"], // this screenshot is flaky in edge
```
Comment (marcellmueller) [inline bciers/apps/registration/e2e/workflows/bceidbusiness/industry_user_admin.spec.ts:121]: This helper function is genius, great idea omitting problematic browsers 🤓

**Review by marcellmueller — APPROVED** [review]
Thanks again for taking this on, you've added some really clever solutions 😍 I think these will make our lives easier going forward!

**Pull request merged**